src/Polly/Caching/IAsyncCacheProvider.cs

```diff
@@ -16,7 +16,7 @@ public interface IAsyncCacheProvider
     /// A <see cref="Task{TResult}" /> promising as Result a tuple whose first element is a value indicating whether
     /// the key was found in the cache, and whose second element is the value from the cache (null if not found).
     /// </returns>
-    Task<(bool, object?)> TryGetAsync(string key, CancellationToken cancellationToken, bool continueOnCapturedContext);
+    Task<(bool CacheHit, object? Result)> TryGetAsync(string key, CancellationToken cancellationToken, bool continueOnCapturedContext);
```

Comment:
We can't make changes like this to the public interface. This is why the build is failing, because the public API no longer matches the baseline.

For public members, the warning just has to be suppressed. Otherwise it would be a breaking change for users.

Reply:
Really. I've done something stupid